project/MimaExcludes.scala

```diff
@@ -118,6 +118,9 @@ object MimaExcludes {
     // [SPARK-26632][Core] Separate Thread Configurations of Driver and Executor
     ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.network.netty.SparkTransportConf.fromSparkConf"),
 
+    // [SPARK-16872][ML][PYSPARK] Impl Gaussian Naive Bayes Classifier
+    ProblemFilters.exclude[DirectMissingMethodProblem]("org.apache.spark.ml.classification.NaiveBayesModel.this"),
```

Comment:
Can we keep this constructor, if it's easy? It relates to the question of subclassing I guess.

Reply:
If we do not make a new subclass, then I guess we can not keep it.
However, this constructor is not exposed to end users.